cudamapper/include/claragenomics/cudamapper/index.hpp

```diff
@@ -121,10 +121,10 @@ class Index
                  const cudaStream_t cuda_stream   = 0);
 };
 
-/// IndexHostCopy - Creates and maintains a copy of computed IndexGPU elements on the host, then allows to retrieve target
+/// IndexHostCopyBase - Creates and maintains a copy of computed IndexGPU elements on the host, then allows to retrieve target
 /// indices from host instead of recomputing them again
 ///
-class IndexHostCopy
+class IndexHostCopyBase
```

Comment:
Not your mistake, but while you are at it: could you add a virtual destructor to the base class?

Reply:
done

Reply:
it needs to be `= default;` not `= delete;` :)

Reply:
whoops :)